backend/routes/courses.js

```diff
@@ -298,4 +298,45 @@ router.post("/updateCourse", async (req, res, next) => {
   }
 });
 
+/**
+ *
+ * @api {post} /assignCourseToCurriculum
+ * @apiDescription assign a course to a curriculumType (core, techComp, complementaries)
+ * @apiParam (body) {string} curriculumType, {string} courseCode, {string} curriculum
+ * @apiExample {curl} Example usage:
+ *	curl -X POST \
+ *  -H 'Content-Type: application/json' \
+ *  -d '{"curriculumType": "core", "courseCode": "ECSE 428", "curriculum": "ee"}' \
+ *  http://localhost:3001/courses/assignCourseToCurriculum
+ *
+ * @returns True on success
+ *          curriculumn curriculumType not existes
```

Comment:
```suggestion
 *          curriculum curriculumType not exists
```
Typo on this line and the three following lines (exists* instead of existes and curriculum* instead of curriculumn)